code/device/mainwindow.cpp

```diff
@@ -31,36 +32,42 @@ void main_window::timerEvent(QTimerEvent *event)
     }
 }
 
-void main_window::setNewView(svg_view * toSet)
+void MainWindow::setNewView(QGraphicsSvgItem *toSet)
 {
-    view = toSet;
+    //mapViewer->InitMap(toSet);
 }
 
-bool main_window::LoadFile(const QString &svgFileName, const QString &xmlFileName)
+bool MainWindow::LoadFile(const QString &svgFileName, const QString &xmlFileName)
 {
     if (QFileInfo::exists(svgFileName) && QFileInfo::exists(xmlFileName)) // maybe move checks to view module? - KYG
     {
         mapInfo->SetAnotherMap(svgFileName, xmlFileName);
```

Comment:
may be separate check of svg existing and init from check of xml existing and init